src/Components/StepTwo/StepTwo.js

```diff
@@ -1,8 +1,118 @@
-import React, {useState} from 'react';
+import React, {useState, useEffect} from 'react';
 import './StepTwo.scss';
+import axios from 'axios';
```

Comment:
It's not good practice to call axios in each component. Imagine scenario:
We are using axios, but one day we have a new requirement to use fetch instead. You would need to go through each component that calls axios, and replace the whole fetching logic (and error handling too).
It's easier to abstract it - please see [this article](https://netbasal.com/the-importance-of-abstraction-in-js-ea27e07e996).